packages/devtools_app/lib/src/charts/chart.dart

```diff
@@ -20,35 +35,43 @@ class Chart extends StatefulWidget {
   final ChartController controller;
 
   @override
-  ChartState createState() => ChartState();
+  ChartState createState() => ChartState(controller);
```

Comment:
Needs to be zero argument. State objects can be reused so this is very dangerous to do.

Reply:
Back to original.